lib/widget/cards/reusable_card_base.dart

```diff
@@ -59,7 +95,7 @@ class ReusableCardBase extends StatelessWidget {
         child: Padding(
           padding: padding,
           child: Container(
-            height: height,
+            height: null,
```

Comment:
isn't this the same as not specifying the height at all?

Reply:
Oops, this shouldn't be here at all, good pickup